fedimint-client/src/db.rs

```diff
@@ -38,6 +38,22 @@ pub enum DbKeyPrefix {
     ClientInviteCode = 0x30,
     ClientInitState = 0x31,
     ClientMetadata = 0x32,
+    /// Arbitrary data of the applications integrating Fedimint client and
+    /// wanting to store some Federation-specific data in Fedimint client
+    /// database.
+    ///
+    /// New users are encouraged to use this single prefix only.
+    //
+    // TODO: https://github.com/fedimint/fedimint/issues/4444
+    //       in the future, we should make all global access to the db private
+    //       and only expose a getter returning isolated database.
+    UserData = 0xb0,
+    /// Prefixes between 0xb1-0xff shall all be considered allocated for
+    /// historical and future external use
+    ExternalReservedStart = 0xb1,
+    /// Prefixes between 0xb1-0xff shall all be considered allocated for
+    /// historical and future external use
+    ExternalReservedEnd = 0xff,
```

Comment:
This is exclusive or inclusive? iirc we use 0xff to denote a global database entry (i.e not in a module)

Reply:
I missed that. Ha.

OK. So maybe `0xb0..=0xcf` for external and `0xd0..=0xff` for internal use.

Reply:
Good with me